### PR TITLE
Add workaround for xvfb errors

### PR DESCRIPTION
**Pull request by muizzk**
@cotsog for ZD 6580.

### CODE REVIEW

**Comment by lechiffre10** [comment]
I think the issue here is the gap between Enterprise and Hosted and not necessarily the documentation on the Hosted side ( Display is set properly on Xenial hosted) and we need to raise this in Enterprise Sync. Also maybe @cotsog might want to chime in but we need to find a way to communicate that the Hosted documentation doesn't necessarily apply to the Enterprise side considering the gap?

**Comment by ssabrii** [comment]
I think the issue here is the gap between Enterprise and Hosted and not necessarily the documentation on the Hosted side ( Display is set properly on Xenial hosted) and we need to raise this in Enterprise Sync. Also maybe @cotsog might want to chime in but we need to find a way to communicate that the Hosted documentation doesn't necessarily apply to the Enterprise side considering the gap?

File: user/gui-and-headless-browsers.md [hunk 1]
```diff
@@ -72,6 +72,14 @@ services:
 ```
 {: data-file=".travis.yml"}
 
+> Sometimes it may be necessary to explicitly set the DISPLAY on Travis CI Enterprise. You can achieve this with the following snippet:
+
+```yaml
+before_script:
+  - "export DISPLAY=:99.0"
```
Comment (cotsog) [inline user/gui-and-headless-browsers.md]: Do you know if it's specific to `dist: xenial`? If so, I would put the full recipe:

```yaml
dist: xenial
services:
  - xvfb

before_script:
  - "export DISPLAY=:99.0"
```

Comment (muizzk) [reply user/gui-and-headless-browsers.md]: No idea. @ssabrii please can you confirm? Thanks

**Comment by muizzk** [comment]
@cotsog updating as per discussion. Thanks

**Comment by plaindocs** [comment]
Also does this fully address #2233  ?

File: user/gui-and-headless-browsers.md (hunk 1 above)
Comment (Litlangelok) [reply user/gui-and-headless-browsers.md]: qguz9ZVRYsdtfzwnm6qX

Comment (Litlangelok) [reply user/gui-and-headless-browsers.md]: > Do you know if it's specific to `dist: xenial`? If so, I would put the full recipe:
> 
> ```yaml
> dist: xenial
> services:
>   - xvfb
> 
> before_script:
>   - "export DISPLAY=:99.0"
> ```

Comment (Litlangelok) [reply user/gui-and-headless-browsers.md]: > Do you know if it's specific to `dist: xenial`? If so, I would put the full recipe:
> 
> ```yaml
> dist: xenial
> services:
>   - xvfb
> 
> before_script:
>   - "export DISPLAY=:99.0"
> ```

**Review by Litlangelok — CHANGES_REQUESTED** [review]

**Pull request merged**